source/blender/draw/intern/draw_manager.c

```diff
@@ -3437,10 +3437,14 @@ void DRW_game_render_loop(bContext *C,
 
   const int object_type_exclude_viewport = v3d->object_type_exclude_viewport;
 
-  DRW_hair_init();
+  /* Update UBO's */
+  DRW_globals_update();  
 
   /* Init engines */
   drw_engines_init();
+  DRW_curves_init();
```

Comment:
Is i working if we put that before drw_engines_init?

Reply:
Yes, it works ok. I move them.